src/http/api/views/account.view.ts

```diff
@@ -0,0 +1,331 @@
+import { type Actor, type Collection, isActor } from '@fedify/fedify';
+import { getAccountHandle } from 'account/utils';
+import type { FedifyContextFactory } from 'activitypub/fedify-context.factory';
+import { getHandle } from 'helpers/activitypub/actor';
+import { sanitizeHtml } from 'helpers/html';
+import type { Knex } from 'knex';
+import { lookupAPIdByHandle, lookupObject } from 'lookup-helpers';
+import type { Site } from 'site/site.service';
+import type { AccountDTO } from '../types';
+
+/**
+ * Additional context that can be passed to the view
+ */
+interface ViewContext {
+    /**
+     * Site instance - This will be used to resolve the default site account
+     * and determine if that account is following / followed by the account
+     * being viewed
+     */
+    site?: Site;
```

Comment:
I _think_ we want context to be an `Account` rather than `Site` - otherwise we're leaking the idea of the default account for a site into this view.

If we pass in an account rather than a site, then when we eventually support multiple accounts for a site, the controller can just pass in like `ctx.get('authenticateAccount')` or something like this?

Reply:
Yeh i went with this so we don't have to double query the default account - If you are requesting your own profile, we cant just return this account, we still need to go via the view to then get the rest of the data needed (so 2 queries for similar data). Maybe not a big deal?

Reply:
updated in https://github.com/TryGhost/ActivityPub/pull/505/commits/731f134ca9eb12b9edef9a82908330fc258403fa